lib/get-config.js

```diff
@@ -4,9 +4,25 @@ var path = require('path');
 var assign = require('lodash').assign;
 var existsSync = require('exists-sync');
 var chalk = require('chalk');
-var rules = require('./rules');
+var defaultRules = require('./rules');
+
+var KNOWN_ROOT_PROPERTIES = ['extends', 'rules', 'pending', 'ignore', 'plugins'];
+var KNOWN_EXTENDED_CONFIGURATION = ['recommended'];
+
+function readPluginFromDisk(options, pluginPath) {
+  var providedConfigPath = options.configPath;
+  var configPath = providedConfigPath || path.join(process.cwd(), '.template-lintrc');
```

Comment:
Now that we need this in two places (here and in `readConfigFromDisk`), lets extract it to a shared function:

```js
function resolveConfigPath(options) {
  var configPath = options.providedConfigPath || path.join(process.cwd(), '.template-lintrc');

  try {
    return require.resolve(configPath);
  } catch(e) {
    return null;
  }
}
```